app/src/components/Fields/SelectField.js

```diff
@@ -60,15 +60,20 @@ const SelectField = ({ id, label, options, formik, isRequired }) => {
     if (options.limitByValues) {
       const includeValues =
         options.limitByValues[formik.values[options.limitByField]];
+      // console.log(
+      //   { includeValues },
+      //   options.limitByField,
+      //   options.limitByValues
+      // );
       optionValues = optionValues.filter(
         x => includeValues.indexOf(x.value) !== -1
       );
     }
   }
-
+  // console.log(id, optionValues);
   return (
     <FormControl fullWidth>
-      <InputLabel htmlFor={id} required={isRequired}>
+      <InputLabel htmlFor={id} required={isrequired}>
```

Comment:
```suggestion
      <InputLabel htmlFor={id} required={isRequired}>
```

Reply:
isRequired produces a bunch of errors. Some of those errors were fixed through by adjustment to the dateRange component. Changing to isrequired, removes the errors.

Reply:
```suggestion
      <InputLabel htmlFor={id} required={required}>
```

makes sense that you shouldn't overlap with `isRequired`, but this still needs to change

Reply:
Switched this back, looks like the fixing of spread props on dateRangePicker solves this. See comment below.